khal/calendar_display.py

```diff
@@ -65,6 +110,17 @@ def str_week(week, today):
     for day in week:
         if day == today:
             day = style(str(day.day).rjust(2), reverse=True)
+        elif highlight_event_days:
+            localize = locale['local_timezone'].localize
+            start = localize(datetime.datetime.combine(day, datetime.time.min))
+            end = localize(datetime.datetime.combine(day, datetime.time.max))
+            devents = collection.get_datetime_by_time_range(start, end) + \
+                      collection.get_allday_by_time_range(day)
```

Comment:
it's too far indented now (yes, I know.... sorry about this)
https://travis-ci.org/geier/khal/jobs/83193750
you can also run flake8 on your code in addition to pep8 to see if it will raise any errors